src/test/java/it/de/aservo/atlassian/confapi/rest/AbstractUserResourceFuncTest.java

```diff
@@ -0,0 +1,105 @@
+package it.de.aservo.atlassian.confapi.rest;
+
+import de.aservo.atlassian.confapi.constants.ConfAPI;
+import de.aservo.atlassian.confapi.model.UserBean;
+import org.apache.wink.client.ClientAuthenticationException;
+import org.apache.wink.client.ClientResponse;
+import org.apache.wink.client.Resource;
+import org.junit.Ignore;
+import org.junit.Test;
+
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+public abstract class AbstractUserResourceFuncTest {
+
+    @Test
+    public void testGetUser() {
+        UserBean exampleBean = getExampleBean();
+        Resource usersResource = ResourceBuilder.builder(ConfAPI.USERS + "?userName=" + exampleBean.getUserName()).build();
+
+        ClientResponse clientResponse = usersResource.get();
+        assertEquals(Response.Status.OK.getStatusCode(), clientResponse.getStatusCode());
+
+        UserBean userBean = clientResponse.getEntity(UserBean.class);
+        assertNotNull(userBean);
+    }
+
+    @Test
+    public void testSetUserEmailAddress() {
+        UserBean exampleBean = getExampleBean();
+        Resource usersResource = ResourceBuilder.builder(ConfAPI.USERS + "?userName=" + exampleBean.getUserName()).build();
```

Comment:
Hier nur eine Kleinigkeit: Für `"?userName="` hätte ich lieber eine private Konstante `USERNAME_PARAM` oder so.

Reply:
Noch besser wäre eine globale Konstante für `userName`, auch zur Verwendung in `UserResource`

Reply:
Selbstverständlich, ich habe die Konstante `PARAM_USERNAME` und eine Methode `getUserNameQueryParam(UserBean userBean)` eingebaut, die von allen ableitenden Tests verwendet weren kann